types/color-string/color-string-tests.ts

```diff
@@ -21,10 +21,12 @@ color = colorString.get.hwb('hwb(60, 3%, 60%, 0.6)');
 
 color = colorString.get.rgb('invalid color string');
 
-let nothing: null;
-nothing = colorString.get.rgb(null);
-nothing = colorString.get.hsl(null);
-nothing = colorString.get.hwb(null);
+function testNullableInput(input: string | null) {
+    color = colorString.get.rgb(input);
+}
+
+// $ExpectError
+color = colorString.get.rgb();
```

Comment:
The following tests should test the new functionality:
```suggestion
colorString.get.rgb(null); // $ExpectType null
colorString.get.rgb('hwb(60, 3%, 60%)'); // $ExpectType Color
colorString.get.rgb(null as string | null); // $ExpectType Color | null
colorString.get.rgb(); // $ExpectError

colorString.get.hwb(null); // $ExpectType null
colorString.get.hwb('hwb(60, 3%, 60%)'); // $ExpectType Color
colorString.get.hwb(null as string | null); // $ExpectType Color | null
colorString.get.hwb(); // $ExpectError

colorString.get.hsl(null); // $ExpectType null
colorString.get.hsl('hwb(60, 3%, 60%)'); // $ExpectType Color
colorString.get.hsl(null as string | null); // $ExpectType Color | null
colorString.get.hsl(); // $ExpectError
```